### DISCUSSION TITLE
Problem mit der PU?

### DISCUSSION THREAD

**Discussion by marcusthierfelder** [Digitaler-Impfnachweis/certification-apis#224] [Q&A]
Hallo,

kann es sein das  impfnachweis.info  nicht richtig funktioniert? Wir bekommen von Kunden die Info

GetChallenge to https://id.impfnachweis.info/auth/realms/bmg-ti-certify/protocol/openid-connect/auth?redirect_uri=connector%3A%2F%2Fauthenticated&response_type=code&scope=openid&client_id=user-access-ti&nonce= failed with error 403 and message Forbidden

**Comment by dirkexe** [comment]
Uns liegen auch etliche bestätigte Kunden-Meldungen vor mit dem Fehler "403 Forbidden".
Ich habe hierzu gerade eine E-Mail an dcc-support@ubirch.com gesendet.

**Comment by mge-medisoftware** [comment]
Hallo,
betrifft auch unsere Praxen, ab 6:00 überall 403, letzter erfolgreicher Abruf gestern 19:30

**Comment by thinkberg** [comment]
Das Problem wurde behoben.

**Comment by mge-medisoftware** [comment]
Leider nicht, es hat sich nur gewandelt. Diverse Fehler, häufigste Fehler nun:

Http-Fehler 502, Phase: Authentifikation: challenge holen, Fehler: Bad Gateway
Http-Fehler 401, Phase: Authentifikation: ermittle signierte challenge und öffentliches Zertifikat SMC-B Fehler: Unauthorized
Http-Fehler 404  Phase: Authentifikation: challenge holen Fehler: Not Found

Noch kein Impfzertifikat erfolgreich erstellt.

**Comment by thinkberg** [comment]
Ja, haben wir auch gerade festgestellt. Der IDMP crasht, scheint so, als gibts grad einen massiven login peak.

**Comment by thinkberg** [comment]
Bitte mal pruefen. Wir haben das Problem behoben.

**Reply by ruiyan-yu** [reply]
Klappt bei mir nicht via Konnektor.
Kann den AccessToken nicht abfragen

<html><body>


51: | ^RY(51) | = | "> "
-- | -- | -- | --
52: | ^RY(52) | = | ""
53: | ^RY(53) | = | "{ [5 bytes data]"
54: | ^RY(54) | = | "* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):"
55: | ^RY(55) | = | "{ [130 bytes data]"
56: | ^RY(56) | = | "* Connection state changed (MAX_CONCURRENT_STREAMS == 250)!"
57: | ^RY(57) | = | "} [5 bytes data]"
58: | ^RY(58) | = | ""
59: | ^RY(59) | = | "  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0"
60: | ^RY(60) | = | "  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0"
61: | ^RY(61) | = | "  0     0    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0"
62: | ^RY(62) | = | "  0     0    0     0    0     0      0      0 --:--:--  0:00:04 --:--:--     0"
63: | ^RY(63) | = | "  0     0    0     0    0     0      0      0 --:--:--  0:00:05 --:--:--     0"
64: | ^RY(64) | = | "  0     0    0     0    0     0      0      0 --:--:--  0:00:06 --:--:--     0"
65: | ^RY(65) | = | "  0     0    0     0    0     0      0      0 --:--:--  0:00:07 --:--:--     0"
66: | ^RY(66) | = | "  0     0    0     0    0     0      0      0 --:--:--  0:00:08 --:--:--     0< HTTP/2 502 "
67: | ^RY(67) | = | ""
68: | ^RY(68) | = | "< strict-transport-security: max-age=15552000; includeSubDomains; preload"
69: | ^RY(69) | = | ""
70: | ^RY(70) | = | "< content-type: text/plain; charset=utf-8"
71: | ^RY(71) | = | ""
72: | ^RY(72) | = | "< content-length: 11"
73: | ^RY(73) | = | ""
74: | ^RY(74) | = | "< date: Tue, 01 Feb 2022 11:08:31 GMT"
75: | ^RY(75) | = | ""
76: | ^RY(76) | = | "< "
77: | ^RY(77) | = | ""
78: | ^RY(78) | = | "{ [5 bytes data]"
79: | ^RY(79) | = | ""
80: | ^RY(80) | = | "100    11  100    11    0     0      1      0  0:00:11  0:00:08  0:00:03     2"
81: | ^RY(81) | = | "100    11  100    11    0     0      1      0  0:00:11  0:00:08  0:00:03     2HTTP/2 502 "
82: | ^RY(82) | = | "strict-transport-security: max-age=15552000; includeSubDomains; preload"
83: | ^RY(83) | = | "content-type: text/plain; charset=utf-8"
84: | ^RY(84) | = | "content-length: 11"
85: | ^RY(85) | = | "date: Tue, 01 Feb 2022 11:08:31 GMT"
86: | ^RY(86) | = | ""
87: | ^RY(87) | = | "Bad Gateway"
88: | ^RY(88) | = | "rtn:502"
89: | ^RY(89) | = | "* Connection #0 to host id.impfnachweis.info left intact"


</body>
</html>

**Comment by mge-medisoftware** [comment]
Es werden wieder erfolgreich Zertifikate erstellt!

Recht häufig noch Fehler Http-Fehler 401, Phase: Authentifikation: ermittle signierte challenge und öffentliches Zertifikat SMC-B Fehler: Unauthorized , möglicherweise irgendein Folgefehler...

**Reply by thinkberg** [reply]
Ich wuerde vermuten, dass ein neues Einloggen notwendig ist, damit der Connector ein aktuelles Token erhaelt.

**Reply by ruiyan-yu** [reply]
Es versucht den Endpoint für die Webseite aufzurufen.
Das Problem ist nicht gefixt.

<html><body>


  | ^RY(101) | = | "cache-control: no-store, must-revalidate, max-age=0"
-- | -- | -- | --
102: | ^RY(102) | = | "content-language: en"
103: | ^RY(103) | = | "content-security-policy: frame-src 'self'; frame-ancestors 'self'; object-src 'none';"
104: | ^RY(104) | = | "content-type: text/html;charset=utf-8"
105: | ^RY(105) | = | "date: Tue, 01 Feb 2022 11:08:47 GMT"
106: | ^RY(106) | = | "referrer-policy: no-referrer"
107: | ^RY(107) | = | "strict-transport-security: max-age=15552000; includeSubDomains; preload"
108: | ^RY(108) | = | "x-content-type-options: nosniff"
109: | ^RY(109) | = | "x-frame-options: SAMEORIGIN"
110: | ^RY(110) | = | "x-robots-tag: none"
111: | ^RY(111) | = | "x-xss-protection: 1; mode=block"
112: | ^RY(112) | = | "content-length: 2357"
113: | ^RY(113) | = | ""
114: | ^RY(114) | = | "    <!DOCTYPE html PUBLIC ""-//W3C//DTD XHTML 1.0 Transitional//EN"""
115: | ^RY(115) | = | "            ""http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"">"
116: | ^RY(116) | = | "    <html xmlns=""http://www.w3.org/1999/xhtml"" class=""login-pf"">"
117: | ^RY(117) | = | ""
118: | ^RY(118) | = | "    <head>"
119: | ^RY(119) | = | "        <meta charset=""utf-8"">"
120: | ^RY(120) | = | "        <meta http-equiv=""Content-Type"" content=""text/html; charset=UTF-8""/>"
121: | ^RY(121) | = | "        <meta name=""robots"" content=""noindex, nofollow"">"
122: | ^RY(122) | = | ""
123: | ^RY(123) | = | "                <meta name=""viewport"" content=""width=device-width,initial-scale=1""/>"
124: | ^RY(124) | = | "        <title>Log in to covPass Impfzertifikatservice</title>"
125: | ^RY(125) | = | "        <link rel=""icon"" href=""/auth/resources/iehgc/login/Ubirch-bmg-kv/img/favicon.ico""/>"
126: | ^RY(126) | = | "                <link href=""/auth/resources/iehgc/common/keycloak/node_modules/patternfly/dist/css/patternfly.min.css"" rel=""stylesheet""/>"
127: | ^RY(127) | = | "                <link href=""/auth/resources/iehgc/common/keycloak/node_modules/patternfly/dist/css/patternfly-additions.min.css"" rel=""stylesheet""/>"
128: | ^RY(128) | = | "                <link href=""/auth/resources/iehgc/common/keycloak/lib/zocial/zocial.css"" rel=""stylesheet""/>"
129: | ^RY(129) | = | "                <link href=""/auth/resources/iehgc/login/Ubirch-bmg-kv/css/login.css"" rel=""stylesheet""/>"
130: | ^RY(130) | = | "    <SCRIPT> if (typeof history.replaceState === 'function') {  history.replaceState({}, ""some title"", ""https://id.impfnachweis.info/auth/realms/bmg-ti-certify/login-actions/authenticate?execution=e5b9f6af-f8a6-415b-b215-3c80d31fd743&client_id=user-access-ti&tab_id=EBoxwkHHiHQ""); }</SCRIPT></head>"
131: | ^RY(131) | = | ""
132: | ^RY(132) | = | "    <body class="""">"
133: | ^RY(133) | = | ""
134: | ^RY(134) | = | "    <div class=""header"">"
135: | ^RY(135) | = | "        <img src=""/auth/resources/iehgc/login/Ubirch-bmg-kv/img/logo_kbv.png"" alt=""KV-Logo"">"
136: | ^RY(136) | = | "        <div id=""kc-header-wrapper"""
137: | ^RY(137) | = | "             class="""">covPass Impfzertifikatservice</div>"
138: | ^RY(138) | = | "    </div>"
139: | ^RY(139) | = | ""
140: | ^RY(140) | = | "    <div class=""login-pf-page"">"
141: | ^RY(141) | = | "        <div id=""kc-header"" class=""login-pf-page-header"">"
142: | ^RY(142) | = | "            <div id=""kc-header-wrapper"""
143: | ^RY(143) | = | "                 class="""">covPass Impfzertifikatservice</div>"
144: | ^RY(144) | = | "        </div>"
145: | ^RY(145) | = | "        <div class=""card-pf "">"
146: | ^RY(146) | = | "            <header class=""login-pf-header"">"
147: | ^RY(147) | = | "                        <h1 id=""kc-page-title"">        We are sorry..."
148: | ^RY(148) | = | "</h1>"
149: | ^RY(149) | = | "            </header>"
150: | ^RY(150) | = | "            <div id=""kc-content"">"
151: | ^RY(151) | = | "                <div id=""kc-content-wrapper"">"
152: | ^RY(152) | = | ""
153: | ^RY(153) | = | ""
154: | ^RY(154) | = | "        <div id=""kc-error-message"">"
155: | ^RY(155) | = | "            <p class=""instruction"">Certificate & Signature Verification failed</p>"
156: | ^RY(156) | = | "        </div>"
157: | ^RY(157) | = | ""
158: | ^RY(158) | = | ""
159: | ^RY(159) | = | "                </div>"
160: | ^RY(160) | = | "            </div>"
161: | ^RY(161) | = | ""
162: | ^RY(162) | = | "        </div>"
163: | ^RY(163) | = | "    </div>"
164: | ^RY(164) | = | "    </body>"
165: | ^RY(165) | = | "    </html>"
166: | ^RY(166) | = | ""
167: | ^RY(167) | = | "rtn:401"
168: | ^RY(168) | = | "* Connection #0 to host id.impfnachweis.info left intact"


</body>
</html>

**Reply by nicolaslang1** [reply]
Bei uns geht es leider auch noch nicht.

**Reply by thinkberg** [reply]
Was versucht der Konnektor denn da aufzurufen?

**Reply by ruiyan-yu** [reply]
Authentifizierung via TI SMC-B. Die eigentliche Kommunikation macht cURL. Das Problem ist bekannt und wurde schonmal gefixt

**Reply by marcusthierfelder** [reply]
wir bekommen:
https://id.impfnachweis.info/auth/realms/bmg-ti-certify/login-actions/authenticate?execution=e5b9f6af-f8a6-415b-b215-3c80d31fd743&client_id=user-access-ti&tab_id=USVb9kkUHaU
->
Certificate & Signature Verification failed

**Reply by thinkberg** [reply]
Bitte nochmal pruefen.

**Reply by marcusthierfelder** [reply]
jetzt geht es

**Reply by mge-medisoftware** [reply]
ja, funktioniert

**Reply by marcusthierfelder** [reply]
vielen dank für die hilfe

**Reply by thinkberg** [reply]
Super, Sorry dass es so lange gedauert hat.

**Reply by Ein-Tim** [reply]
@thinkberg Gibt es eine öffentliche Status-Seite oder sowas? Ich denke, das wäre echt hilfreich.

**Reply by marcusthierfelder** [reply]
Die Meldung kam bei der Gematik, wurde aber ab 10:30 als behoben gemeldet
<img width="921" alt="Bildschirmfoto 2022-02-01 um 14 58 25" src="https://user-images.githubusercontent.com/2109844/151987353-f424193f-20a5-4784-adfd-763aef57dd91.png">
.

**Comment by mge-medisoftware** [comment]
Meist Http-Fehler 401 bei Senden signierter challenge und öffentlichem Zertifikat SMC-B (Fehler: Unauthorized)

Tritt auch nach Neustart des PVS-Programmes auf (was einem neuen Login entspricht).

Nur wenige Praxen können erfogreich Impfnachweise erstellen, merkwürdig..
